build-scripts/golang-built-rpms/golang-built-rpms

```diff
@@ -0,0 +1,302 @@
+#!/usr/bin/env bash
+
```

Comment:
Needs `set -euo pipeline` to prevent silent failure on exception.

Reply:
agreed, this should pretty much come standard (even when hacking up a script...)

Reply:
:man_facepalming: of course ... resolved.